feat(content): Hogshead's Dilemma (#9770)
endless-sky/endless-sky: data/human/human missions.txt

			action
				payment 120000
			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`

> mission "Hogshead's Dilemma"
> 	invisible

Reviewer: ```suggestion
mission "Hogshead's Dilemma"
	minor
	invisible
```
Shouldn't this be minor?

	source
		near Sol 1 10
	to offer
		random < 5
	on offer
		conversation
			`While passing through the spaceport, you notice a few energetic pilots gathered around a computer terminal, invested in some sort of conversation. One of the pilots speaks louder than the rest, sometimes getting looks from others in the spaceport. "You've fooled me before, and I reckoned you were this time! And here I thought them dang things were old news!" The pilot lets out a hearty chuckle. "Even nowadays them Firkins are still getting names!" After another minute or two, the group walks away from the terminal and out of your sight.`
			choice
				`	(Go see what this is about.)`
				`	(Ignore it.)`
					decline

> 			`	You walk up to the computer terminal, and on it is displayed a news article with a title that reads "Popular Votes For Ship Name Poll - Unexpected Results." The article is about a small transport company harkening back to older times with a special transport voyage centered around a Hogshead space liner. Hogsheads have all but lost the charm they once had, but it seems a good number of pilots are eager to join in on the fun.`

Reviewer: ```suggestion
			`	You walk up to the computer terminal, and on it is displayed a news article with a title that reads "Popular Votes For Ship Name Poll - Unexpected Results." The article is about a small transport company hearkening back to older times with a special transport voyage centered around a Hogshead space liner. Hogsheads have all but lost the charm they once had, but it seems a good number of pilots are eager to join in on the fun.`
```

> 			`	For the event, the transport company decided to have a public poll to name the ship, and while initial suggestions were casual names such as "Stark Lady" or "Timelight", some of the most popular ones are actually a collection of joke names - ones you would imagine a toddler would have made up. Apparently, the names have gotten enough attention and enjoyment out of pilots that the company decided to take the top five most popular results and hold a vote on them.`

Reviewer: ```suggestion
			`	For the event, the transport company decided to have a public poll to name the ship, and while initial suggestions were casual names such as "Stark Lady" or "Timelight", some of the most popular ones are actually a collection of joke names - ones you would imagine a toddler would have come up with. Apparently, the names have gotten enough attention and enjoyment out of pilots that the company decided to take the top five most popular results and hold a vote on them.`
```
Soft

> 			`	Would you like to vote on the name as well?`

Reviewer 2: just an idea but I think it could be nice if a thumbnail of a Hoshead was displayed, so the player knows what ship this is talking about?
it takes a bunch of time to find one even if you want to, in CCOR space so I wouldnt expect most ppl to know what ship we're talking about otherwise

			choice
				`	(Yes.)`
				`	(No.)`
					decline
			`	You navigate the article until you are directed to the poll itself, presenting you with the top five options.`
			choice
				`	Star Empress`
				`	Tycho's Legacy`

> 				`	ShippyMcShipFace`

Reviewer: ```suggestion
				`	Shippy McShipface`
```
I thought we would be following the example of "Boaty McBoatface". The capitalization of ShippyMcBoatFace doesn't make much sense imo

				`	It's Dark Here`
				`	What Asteroid`
			action
				event "hogshead poll" 7
			`	After you made your selection, a "thank you" message appears, telling you that the result of the poll will be broadcast on the news once the voting period has finished. Afterwards, you go back about your business.`
				decline

event "hogshead poll"

mission "Hogshead's Dilemma II"
	invisible
	source
		near Sol 1 10
	to offer
		has "Hogshead's Dilemma: offered"
		has "event: hogshead poll"
	on offer
		clear "hogshead poll"
		conversation

> 			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh, while others seem disappointed, or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - ShippyMcShipFace. The sheer ridiculousness of the situation is capturing the attention of almost anyone who catches a glimpse of the monitor. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`

Reviewer: ```suggestion
			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh while others seem disappointed or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - Shippy McShipface. The sheer ridiculousness of the situation is capturing the attention of almost anyone who catches a glimpse of the monitor. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`
```

> 			`	You recall that, back in their day, Hogsheads were sort of a business fad thanks to a ton of discussion and nicknames based on the shape of their engine pods. In turn, this gave them a brief but strong popularity amongst organizations looking to grab the average passenger's attention. In the end, the transport company responsible for the naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back. Perhaps the name "ShippyMcShipFace" is fitting after all.`

Reviewer: ```suggestion
			`	You recall that, back in their day, Hogsheads were sort of a business fad thanks to a ton of discussion and nicknames based on the shape of their engine pods. In turn, this gave them a brief but strong popularity amongst organizations looking to grab the average passenger's attention. In the end, the transport company responsible for the naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back. Perhaps the name "Shippy McShipface" is fitting after all.`
```

> 			`	You recall that, back in their day, Hogsheads were sort of a business fad thanks to a ton of discussion and nicknames based on the shape of their engine pods. In turn, this gave them a brief but strong popularity amongst organizations looking to grab the average passenger's attention. In the end, the transport company responsible for the naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back. Perhaps the name "ShippyMcShipFace" is fitting after all.`

Reviewer: Are you sure about First Last **recalling** this information, which is being introduced to the player for the first time? Would the character have any reason to know that?

Author: If the Hogshead was such a fad/meme back in the day, it makes sense that it would be on historical channels or documents that the player gets involved in as an aspiring pilot. Same with if a pilot today learned of the A-10 or something similar. The player is also said to already have knowledge of the Quarg from media, etc., prior to meeting them, so I feel like this is within reason.

			action
				payment 120000
			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`

mission "Hogshead's Dilemma"
	minor
	invisible
	source
		near Sol 1 10
	to offer
		random < 5
	on offer
		conversation

> 			`While passing through the spaceport, you notice a few energetic pilots gathered around a computer terminal, invested in some sort of conversation. One of the pilots speaks louder than the rest, sometimes getting looks from others in the spaceport. "You've fooled me before, and I reckoned you were this time! And here I thought them dang things were old news!" The pilot lets out a hearty chuckle. "Even nowadays them Firkins are still getting names!" After another minute or two, the group walks away from the terminal and out of your sight.`

Reviewer 3: I must be out of touch - what is a `Firkin`?

Reviewer 4: ~~9 gallons~~

Author: The Hogshead's name is a synonym of Keg, which the engines look like. Firkin is also a synonym for that. Both were options for the ship's name, but the former was chosen, so throwing the latter here as another "nickname" for it, and the ship's lore is all about nicknames.

Reviewer 3: Ah, gotcha. Makes sense!

			action
				payment 120000
			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`

mission "Hogshead's Dilemma"
	invisible
	source
		near Sol 1 10
	to offer
		random < 5
	on offer
		conversation
			`While passing through the spaceport, you notice a few energetic pilots gathered around a computer terminal, invested in some sort of conversation. One of the pilots speaks louder than the rest, sometimes getting looks from others in the spaceport. "You've fooled me before, and I reckoned you were this time! And here I thought them dang things were old news!" The pilot lets out a hearty chuckle. "Even nowadays them Firkins are still getting names!" After another minute or two, the group walks away from the terminal and out of your sight.`
			choice
				`	(Go see what this is about.)`
				`	(Ignore it.)`
					decline

> 			`	You walk up to the computer terminal, and on it is displayed a news article with a title that reads 'Popular Votes For Ship Name Poll - Unexpected Results'. The article is about a small transport company harkening back to older times with a special transport voyage centered around a Hogshead space liner. Hogsheads have all but lost the charm they once had, however it seems a good amount of pilots are eager to join in on the fun.`

Reviewer 5: ```suggestion
			`	You walk up to the computer terminal, and on it is displayed a news article with a title that reads "Popular Votes For Ship Name Poll - Unexpected Results." The article is about a small transport company harkening back to older times with a special transport voyage centered around a Hogshead space liner. Hogsheads have all but lost the charm they once had, but it seems a good number of pilots are eager to join in on the fun.`
```
Pretty sure this is the way that them quotation marks are used in Yankland.

I believe "number" is preferred instead of "amount" when it comes to countable objects.

			`	For the event, the transport company decided to have a public poll to name the ship, and while initial suggestions were casual names such as "Stark Lady" or "Timelight", some of the most popular ones are actually a collection of joke names - ones you would imagine a toddler would have made up. Apparently, the names have gotten enough attention and enjoyment out of pilots that the company decided to take the top five most popular results and hold a vote on them.`
			`	Would you like to vote on the name as well?`
			choice
				`	(Yes.)`
				`	(No.)`

> 				decline

Reviewer 5: ```suggestion
					decline
```

			`	You navigate the article until you are directed to the poll itself, presenting you with the top five options.`
			choice
				`	Star Empress`
				`	Tycho's Legacy`
				`	ShippyMcShipFace`

> 				`	Its Dark Here`

Reviewer 5: ```suggestion
				`	It's Dark Here`
```
Not sure if the missing apostrophe is intentional or not?

Author: It's intentional, but I get the feeling it will end up a typo to many, so I guess this is fine.

				`	What Asteroid`
			action

> 				set "hogshead poll start"

Reviewer 5: Why not just set the event here?

Author: Is it `set `event...` or just `event...` ?

Reviewer 5: Just `event "hogshead poll" 7` I believe.

> 			`	Once you have made your selection, a 'thank you' message appears. The result of the poll will be broadcasted on the news once the voting period is over. Afterwards, you go back about your business.`

Reviewer 5: ```suggestion
			`	After you made your selection, a "thank you" message appears, telling you that the result of the poll will be broadcast on the news once the voting period has finished. Afterwards, you go back about your business.`
```

				decline

mission "Hogshead's Dilemma Poll"
	invisible
	to complete
		never
	to offer
		has "Hogshead's Dilemma: offered"
		has "hogshead poll start"
	on offer
		fail
		event "hogshead poll" 7

event "hogshead poll"

mission "Hogshead's Dilemma II"
	invisible
	source
		near Sol 1 10
	to offer
		has "Hogshead's Dilemma: offered"
		has "event: hogshead poll"
	on offer
		clear "hogshead poll start"
		clear "hogshead poll"
		conversation
			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh, while others seem disappointed, or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - ShippyMcShipFace. The sheer ridiculousness of the situation is capturing the attention of almost anyone who catches a glimpse of the monitor. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`

> 			`	You recall that back in the day Hogsheads were a sort of a business fad thanks to a ton of discussion and nicknames based on the shape of their engine pods. In turn, this gave them a brief but strong popularity amongst organizations looking to grab the average passenger's attention. In the end, the transport company responsible for the naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back. Perhaps the name 'ShippyMcShipFace' is fitting after all.`

Reviewer 5: ```suggestion
			`	You recall that, back in their day, Hogsheads were sort of a business fad thanks to a ton of discussion and nicknames based on the shape of their engine pods. In turn, this gave them a brief but strong popularity amongst organizations looking to grab the average passenger's attention. In the end, the transport company responsible for the naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back. Perhaps the name "ShippyMcShipFace" is fitting after all.`
```
I dunno what Americans are on about with their quotation marks.

Seems like "their" would be better than "the" here?
